egs/grid/VSR/install.sh

```
@@ -0,0 +1,55 @@
+
+  cd "$AVHUBERT_DIR"
+  git submodule init
+  git submodule update
+
+  cd fairseq
+  pip install --editable ./
+fi
+
+
+
+cd $ICEFALL_DIR/egs/grid/VSR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)